src/main/webapp/scripts/map.js

```diff
@@ -26,14 +55,40 @@ function initMap() {
   map.addListener("click", e => {
     placeMarkerAndPanTo(e.latLng, map);
   });
+  /*Add Hard-Coded Markers */
+  addMarker(map);
+}
+
+/* Temporary Function to Hold Marker for Sky Garden */
+function addMarker(map) {
+  var latLng = {lat: 51.510881, lng: -0.083751}
```

Comment:
can you change this to const
and put the details as the variable name, e.g. const skyGarden =